python/ray/tune/syncer.py

```diff
@@ -458,23 +459,34 @@ def sync_down_if_needed(
 
     def wait_or_retry(self, max_retries: int = 3, backoff_s: int = 5):
         assert max_retries > 0
-        last_error = None
-        for _ in range(max_retries):
+        last_error_traceback = None
+        for i in range(max_retries + 1):
```

Comment:
So this makes semantically sense (we try 4 times --> thus 3 retries), but let's set default `max_retries` to 2 then